src/Malt.jl

```diff
@@ -102,9 +102,9 @@ mutable struct Worker <: AbstractWorker
 
     stdout::Pipe
     stderr::Pipe
-    function Worker(; env=String[], exeflags=[])
+    function Worker(; env=String[], exename=exe=Base.julia_cmd()[1], exeflags=[])
```

Comment:
@vchuravy what does this syntax do? `exename=exe=Base.julia_cmd()[1]`

Is it on purpose or is it a typo?